yamcs-api/src/main/proto/yamcs/protobuf/filetransfer/filetransfer.proto

```diff
@@ -208,16 +233,13 @@ message CreateTransferRequest {
   // **Required** The path at the remote entity. Example: ``a/local/path/some_filename``.
   optional string remotePath = 5;
   
-  optional DownloadOptions downloadOptions = 6;
+  optional UploadOptions uploadOptions = 6;
   
-  // Configuration options specific to ``UPLOAD`` transfers.
-  optional UploadOptions uploadOptions = 7;
-
   //used to derive the source entity id
-  optional string source = 9;
+  optional string source = 7;
 
   //used to derive the destination entity id
-  optional string destination = 10;
+  optional string destination = 9;
```

Comment:
Could we keep this at 10?
(Renumbering will break backwards compatibility in Python client)